frontend/src/pages/CourseSelector/CourseMenu/CourseTitle/CourseTitle.tsx

```diff
@@ -1,17 +1,11 @@
 import React from 'react';
-import { useDispatch, useSelector } from 'react-redux';
 import {
```

Comment:
can we put this entire import statement into this into one line now? Or will eslint complain 👀